### PR TITLE
Use `django.db.models.BigAutoField` as default auto field

### LINKED ISSUE #137: Linkcheck does not define a `default_auto_field`
Linkchecks's [LinkcheckConfig](https://github.com/DjangoAdminHackers/django-linkcheck/blob/a6599a61c11026c3fb394ba9f213a21fea111d30/linkcheck/apps.py#L45) does not define a `default_auto_field`, which means it inherits the config from the project it's being used in, or from Django's default.

The downside of this config is that if the project changes its default auto field, it can not easily generate database migrations for the linkcheck app. So users are confronted with the decision of:
- [overwriting the linkcheck config](https://github.com/digitalfabrik/integreat-cms/blob/43990f92bee917e64d619c49a0f67678fba9c469/integreat_cms/linkcheck/apps.py#L7-L12),
- [monkeypatching their app label](https://github.com/digitalfabrik/integreat-cms/blob/43990f92bee917e64d619c49a0f67678fba9c469/integreat_cms/cms/migrations/0050_increase_max_url_length.py#L38-L41) to be able to add linkcheck migrations in their own directory,
- or just running `django-admin makemigrations linkcheck` on the production system itself (and having to merge these individual migrations with the upstream ones in case we add more migrations)

All of these approaches don't seem ideal to me, so I'd suggest we should decide on one auto field we're using for our models and provide a migration for it.

### PR DESCRIPTION
**Pull request by timobrembeck**
Fixes #137 (see issue description for context info)

### CODE REVIEW

File: linkcheck/migrations/0005_default_big_auto_field.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (claudep) [inline linkcheck/migrations/0005_default_big_auto_field.py]: Wow.... Python 2 resurrected :rofl:

**Review by claudep — APPROVED** [review]
I think this makes sense!

File: linkcheck/migrations/0005_default_big_auto_field.py (hunk 1 above)
Comment (timobrembeck) [reply linkcheck/migrations/0005_default_big_auto_field.py]: just copied + pasted from the other ones :laughing: 
should I remove all of them?

Comment (claudep) [reply linkcheck/migrations/0005_default_big_auto_field.py]: Oh sure, absolutely!

File: linkcheck/migrations/0005_alter_link_id_alter_url_id.py [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-            ),
-        ),
-        migrations.AlterField(
-            model_name="url",
-            name="id",
-            field=models.BigAutoField(
-                auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
-            ),
-        ),
-    ]
```
Comment (timobrembeck) [inline linkcheck/migrations/0005_alter_link_id_alter_url_id.py]: Sorry, somehow during the rebase of the preview PR I missed this auto-generated migration :unamused: 
I'm just renaming it in this PR, ok?

File: linkcheck/migrations/0001_initial.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 from __future__ import unicode_literals
```
Comment (timobrembeck) [inline linkcheck/migrations/0001_initial.py]: Should I remove this legacy import as well? :thinking:

Comment (claudep) [reply linkcheck/migrations/0001_initial.py]: Yes, please!

**Comment by claudep** [comment]
Would be nice if the cleanup could be done in a separate commit.

**Pull request merged**